mscore/palettebox.cpp

```diff
@@ -63,6 +69,18 @@ PaletteBox::PaletteBox(QWidget* parent)
       hlSearch->setContentsMargins(5,0,5,0);
       hlSearch->addWidget(_searchBox);
 
+
+      QFile file(":/default.txt");
+      file.open(QIODevice::ReadOnly);
+      TreeModel model(file.readAll());
+      file.close();
+
+      QTreeView view;
```

Comment:
Same problem here: the TreeView doesn't appear because it is destroyed as soon as the function goes out of scope. In this case there is no need to give the object a parent because [this is done for you](https://doc.qt.io/qt-5/layout.html#tips-for-using-layouts) later when you add the view to the layout. This means it is safe to do:

```c++
QTreeView* view = new QTreeView;
```

This was not true for the model, because [adding a model to a view does not make it a child of that view](https://doc.qt.io/qt-5/qabstractitemview.html#setModel). This is because the model might need to be shared between multiple views, so you don't want it to be deleted when one of the views is destroyed.

Note: don't forget to find-and-replace all occurances of `view.` with `view->` to make it compile.